Unity-Technologies/Graphics
# Use color temperature in the player for Enlighten real-time GI

### Purpose of this PR
Fixes Correlated Color Temperature not being applied in Player builds for Enlighten real-time GI lights (case 1370438).

### Testing status
Tested locally on Windows 10 using the repro from the case in editor and standalone player:
1. Download and open CornellBox project. Use an appropriate branch for your RP of choice.
2. Open CornellBox scene
3. Select either the Directional Light, Spotlight, or Point Light game object
4. Make sure that it is using any color temperature value
5. Set its mode to Realtime
6. Enable Realtime GI in the Lighting window and bake
7. Build a standalone build
8. Notice that the light's color is white, regardless of the color temperature setting

I have tested editor and standalone player using BiRP, URP and HDRP.

Screenshot of the editor showing a light source with a low color temperature and a white color filter https://drive.google.com/file/d/1qdFfk63dUQPjpbzIionGfTHFwezCI0bm/view?usp=sharing.

Screenshot of a standalone player build of the same scene in BiRP https://drive.google.com/file/d/1bp0l8R_2phwegRWuIZuEH2KKE0BM-hOG/view?usp=sharing.

Screenshot of a standalone player build of the sample scene in URP with a realtime GI light with color temperature https://drive.google.com/file/d/1DLnEyRDdSsWX0M2mvU89llI8cklTNDnh/view?usp=sharing 

Screenshot of a standalone player build of the sample scene in HDRP with a realtime GI light with color temperature https://drive.google.com/file/d/1ik9LgFBkrFY2sjqYWUiMw64QXmF_n9WO/view?usp=sharing

Suggestions for manual testing: HDRP and URP sanity checks for lighting and realtime GI and baked GI.

### Comments to reviewers
Native code PR for BiRP https://ono.unity3d.com/unity/unity/pull-request/136092/_/lighting/bugfix/1370438
